### ISSUE TITLE
Ignores command line arguments eg -W 448 -H 448 --steps 15 --fps 15

### ISSUE THREAD

**Issue by G-force78** [TMElyralab/MusePose#18]
Output video at 768x768, script defaults in test_stage_2.py

**Comment by brad12d3** [comment]
I am curious about this too. I am using a resolution of H 960 W 512 and it is definitely processing at that resolution but then seems to downscale it so that the height is 768, so my new resolution becomes H 768 W 410.

So there seems to be a scaling process when the video is finalized that takes it back to a 768 resolution but I'm not sure how to change that.

**Comment by czk32611** [comment]
Hello, setting W and W will result in generating the video at W x H first, and then resize it back to the original size of the pose images. Please check the [code](https://github.com/TMElyralab/MusePose/blob/main/test_stage_2.py#L203).

We also update README accordingly.

**Comment by ManuelMultiverse** [comment]
Hi, this is happening to me, my input video is 1080 x 1920 and the generated pose is 448 x 960 so it is cropping the character.

**Comment by G-force78** [comment]
> Hello, setting W and W will result in generating the video at W x H first, and then resize it back to the original size of the pose images. Please check the [code](https://github.com/TMElyralab/MusePose/blob/main/test_stage_2.py#L203).
> 
> We also update README accordingly.

Right ok, so you use a 768 pose size for a sd1.5 generation, seems odd when the default resolution for 1.5 is 512.
I have looked here to see where that can be changed but can't find anything https://github.com/TMElyralab/MusePose/tree/main/pose/script
Is this where? https://github.com/TMElyralab/MusePose/blob/97ad19e699a314186939c4641a156f34f790d617/pose_align.py#L477

**Comment by brad12d3** [comment]
> Hello, setting W and W will result in generating the video at W x H first, and then resize it back to the original size of the pose images. Please check the [code](https://github.com/TMElyralab/MusePose/blob/main/test_stage_2.py#L203).
> 
> We also update README accordingly.

Yeah, I figured out that the end resolution was based on the generated dwpose video. Is there a way to have the generated align DWpose video keep the same resolution as the original video? I have both my video and reference image the same resolution but the dwpose generated align video is a very different resolution and can crop into the image.

**Comment by G-force78** [comment]
> > Hello, setting W and W will result in generating the video at W x H first, and then resize it back to the original size of the pose images. Please check the [code](https://github.com/TMElyralab/MusePose/blob/main/test_stage_2.py#L203).
> > We also update README accordingly.
> 
> Right ok, so you use a 768 pose size for a sd1.5 generation, seems odd when the default resolution for 1.5 is 512. I have looked here to see where that can be changed but can't find anything https://github.com/TMElyralab/MusePose/tree/main/pose/script Is this where?
> 
> https://github.com/TMElyralab/MusePose/blob/97ad19e699a314186939c4641a156f34f790d617/pose_align.py#L477

Have tested and can confirm you can change the size of the pose video here which aligns it better with the native SD 1.5 resolution.
